packages/graphiql-react/src/editor/tabs.ts

```diff
@@ -5,10 +5,25 @@ import debounce from '../utility/debounce';
 import { CodeMirrorEditorWithOperationFacts } from './context';
 import { CodeMirrorEditor } from './types';
 
+export type TabDefinition = {
+  /**
+   * The contents of the query editor of this tab.
+   */
+  query: string | null;
+  /**
+   * The contents of the variable editor of this tab.
+   */
+  variables: string | null;
+  /**
+   * The contents of the headers editor of this tab.
+   */
+  headers: string | null;
```

Comment:
is there a reason `headers?: string` isn't used instead? 

so, when specifying this, users will need to provide `{ query: '{ id }', variables: null, headers: null }` if they only want to specify a query. I think allowing it to be undefined is preferrable over requiring null here, what do you think @thomasheyenbrock  or @jonathanawesome ?